Sprint-1/implement/max.js

```diff
@@ -2,3 +2,17 @@ function findMax(elements) {
 }
 
 module.exports = findMax;
+// Filter the array to only include numeric values
+const numericElements = elements.filter((item) => typeof item === "number");
+
+// Return -Infinity if the array is empty
+if (numericElements.length === 0) {
+  return -Infinity;
+}
+
+// Find and return the maximum number in the numeric elements
+return Math.max(...numericElements);
+}
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function return the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax(["123", 45])
findMax([0, NaN, 1])
```